academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Inter-Blockchain Communication Introduction"
+order: 1
+description: IBC Protocol introduction
+tag: fast-track
+---
+
+# Introduction: What is IBC?
+
+The [(Inter-Blockchain Communication (IBC)](https://ibcprotocol.org/) Protocol is _a protocol to handle authentication and transport of data between two blockchains_. IBC **requires a minimal set of functions**, specified in the [Interchain Standards (ICS)](https://github.com/cosmos/ibc/tree/master/spec/ics-001-ics-standard). Notice that those specifications do not limit the network topology or consensus algorithm, so IBC can be used with a wide range of blockchains or state machines. IBC is intended to work in execution environments where participants do not necessarily trust each other.
+
+<HighlightBox type="info">
+
+IBC solves a widespread problem, namely cross-chain communication. This problem exists on public blockchains when exchanges wish to perform swaps. The problem emerges earlier in the case of application-specific blockchains where every asset is likely to emerge from its own purpose-built chain. Cross-chain communication is also a challenge in the world of private blockchains in cases where communication with a public chain or other private chains is desirable. There are already [IBC implementations for private blockchains such as Hyperledger Fabric and Corda](https://www.hyperledger.org/blog/2021/06/09/meet-yui-one-the-new-hyperledger-labs-projects-taking-on-cross-chain-and-off-chain-operations).
+
+</HighlighBox>
+
+## Internet of blockchains
+
+IBC is essential for application-specific blockchains like Cosmos. It offers a standard communication channel for applications on two different chains that need to communicate with each other.
```

Comment:
... application-specific blockchains like Cosmos => ... like those in the Cosmos network